LICENSE

```diff
@@ -201,3 +201,10 @@
    limitations under the License.
 
 
+Copyright <YEAR> <COPYRIGHT HOLDER>
```

Comment:
What does this copyright notice refer to exactly?

Reply:
Hi Michiel: This is an appendix to the generic Apache-2.0 license (actually where it tells you how to apply the license to code).  The NOTICE file is where explicit values are filled in.  We agreed this approach with Comcast Legal because the alternative meant the duplication of the text of the license for each copyright holder. - Martin